careers/settings.py

```diff
@@ -31,6 +32,13 @@
 
 ALLOWED_HOSTS = config('ALLOWED_HOSTS', cast=Csv())
 
+# Add AWS internal IP to allowed hosts for ELB health check to work.
+try:
+    EC2_PRIVATE_IP = requests.get('http://169.254.169.254/latest/meta-data/local-ipv4',
```

Comment:
Interesting. Do you think we should do this for our other sites? We've been just setting it to `"*"` I believe.